src/controllers/citiesController.mjs

```diff
@@ -0,0 +1,15 @@
+import { readFileAsync } from "../utils/fileUtils.mjs";
+import fs from "fs/promises";
+import { BASEDIR } from "../../config.mjs";
+
+export const loadCities = async () => {
```

Comment:
Lets adjust our naming to the http like naming convention
loadCities -> getCities